### PR TITLE
{math}[foss/2020b,intel/2020b] MUMPS v5.3.5

### PR DESCRIPTION
**Pull request by SimonPinches**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by casparvl** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/m/MUMPS/MUMPS-5.3.5-foss-2020b-parmetis.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+versionsuffix = '-parmetis'
+
+homepage = 'https://graal.ens-lyon.fr/MUMPS/'
+description = "A parallel sparse direct solver"
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['http://mumps.enseeiht.fr/']
+sources = ['%(name)s_%(version)s.tar.gz']
```
Comment (casparvl) [inline easybuild/easyconfigs/m/MUMPS/MUMPS-5.3.5-foss-2020b-parmetis.eb:12]: Aren't the patches from [here](https://github.com/easybuilders/easybuild-easyconfigs/blob/7c4cd2735f43c34c93355f01f9d5a2f93ac218ef/easybuild/easyconfigs/m/MUMPS/MUMPS-5.3.5-intel-2020b-metis.eb#L14) also relevant for this build against ParMETIS?

File: easybuild/easyconfigs/m/MUMPS/MUMPS-5.3.5-foss-2020b-parmetis.eb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+name = 'MUMPS'
+version = '5.3.5'
+versionsuffix = '-parmetis'
```
Comment (casparvl) [inline easybuild/easyconfigs/m/MUMPS/MUMPS-5.3.5-foss-2020b-parmetis.eb:3]: I know nothing about MUMPS, Metis and Parmetis, but I noticed that this is the first MUMPS EasyConfig that was build against Parmetis. Can you enlighten me a bit: why would one build with one or the other? It seems a bit surprising that no one build against Parmetis before

**Review by casparvl — COMMENTED** [review]
A few questions, not sure if they require adaptations though, maybe not :)

**Comment by casparvl** [comment]
Hm, I tried to build this, but it wouldn't download the sources. Might just be a hickup, might be they moved... @SimonPinches does it still work for you (e.g. if you try to do `--fetch --force-download`)?

File: easybuild/easyconfigs/m/MUMPS/MUMPS-5.3.5-foss-2020b-parmetis.eb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+homepage = 'https://graal.ens-lyon.fr/MUMPS/'
+description = "A parallel sparse direct solver"
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['http://mumps.enseeiht.fr/']
```
Comment (casparvl) [inline easybuild/easyconfigs/m/MUMPS/MUMPS-5.3.5-foss-2020b-parmetis.eb:11]: Needs to be updated, sources are now at https://graal.ens-lyon.fr/MUMPS/ I think

**Comment by boegel** [comment]
closing this since `foss/2020b` is deprecated, see https://docs.easybuild.io/policies/toolchains

**Pull request closed without merging**